packages/arcs/src/arc_labels/canvas.ts

```diff
@@ -2,6 +2,15 @@ import { CompleteTheme } from '@bitbloom/nivo-core'
 import { DatumWithArcAndColor } from '../types'
 import { ArcLabel } from './useArcLabels'
 
+function intersects(a: DOMRect, b?: DOMRect): boolean {
```

Comment:
This function is copied to another couple of places - not sure if there is a good place for such utiiities in nivo?